addon/components/pix-tooltip.hbs

```diff
@@ -11,8 +10,7 @@
         {{if @isLight "pix-tooltip__content--light"}}
         {{if @isWide "pix-tooltip__content--wide"}}"
     >
-      {{this.text}}
+      {{yield to="tooltip"}}
```

Comment:
**question**
C'est quoi le `to="tooltip"` je connais pas :o ? 
C'est normal qu'on ait 2 fois le `{{yield}}` du coup ? Je m'attendais à voir un `{{else}}` avec le deuxième `{{yield}}` :o

Reply:
merci pour les infos 🤗